worker/multi_tenancy_ee.go

```diff
@@ -39,43 +34,12 @@ func (w *grpcWorker) DeleteNamespace(ctx context.Context,
 }
 
 func ProcessDeleteNsRequest(ctx context.Context, ns uint64) error {
-	// Update the membership state to get the latest mapping of groups to predicates.
-	if err := UpdateMembershipState(ctx); err != nil {
-		return errors.Wrapf(err, "Failed to update membership state while deleting namesapce")
-	}
-
-	state := GetMembershipState()
-	g := new(errgroup.Group)
-
-	for gid := range state.Groups {
```

Comment:
Let's not route the delete request through zero.
We can process the delete namespace request in alpha and then return it to zero.

Reply:
Done.